### ISSUE TITLE
Adds Laminas support

### ISSUE THREAD

**Issue by michalbundyra** [phpro/zf-doctrine-hydration-module#45]
As Zend Framework is migrated now to Laminas and we have also released DoctrineModule 3.0 with Laminas support we need migrate also this library.

**Comment by pmarko** [comment]
Any update on this. Is there a plan to add compatibility with doctrine module 4 ?

**Comment by TomHAnderson** [comment]
I agree this should be moved to Laminas.  I propose a new repository named `phpro/doctrine-hydration-module` with support for Laminas.  I'm willing to do the conversion if you'd create the new repo from master branch of this repo.

If this is a project you'd like to no longer support let me know and I'll solve the problem with API Skeletons.

**Comment by veewee** [comment]
Hello @TomHAnderson 

We are no longer using Apigility at PHPro anymore.
Therefor, it is hard for me to make the correct decisions for this package. 

Feel free to use this package as a base for a new laminas package. Once that one is ready, we can archive this one.

Let me know if there is something I can do to make it easier to hand over the project.

**Comment by javabudd** [comment]
Any update on this? It's preventing Laminas applications using api-tools + doctrine from being updated to PHP8

**Comment by TomHAnderson** [comment]
I made the decision to no longer use api-tools + doctrine in my work.  Laminas has not shown an interest in maintaining the doctrine integration.

**Comment by javabudd** [comment]
I have forked the repository and am working on a Laminas + PHP 7/8 compatible implementation. I am willing to port this over to laminas repos anytime.

https://github.com/javabudd/zf-doctrine-hydration-module
